pkg/storage/tpr/install_types.go

```diff
@@ -34,9 +36,23 @@ var thirdPartyResources = []v1beta1.ThirdPartyResource{
 	serviceBindingTPR,
 }
 
+//InstallerTPRs takes in a client and exposes method for installing third party resources
+type InstallerTPRs struct {
```

Comment:
1.  The name of this type and the name in the comment do not match.
2.  Call this `TPRInstaller`, please